### PR TITLE
feat: lend dropdown mobile update

### PR DESCRIPTION
**Pull request by egoroza**
updates mobile view to use the "Categories" dropdown instead of "Explore loans"

### CODE REVIEW

File: src/components/WwwFrame/TheHeader.vue [hunk 1]
```diff
@@ -112,33 +112,34 @@
 						</div>
 
 						<template v-if="lendMenuButtonExp">
```
Comment (roger-in-kiva) [inline src/components/WwwFrame/TheHeader.vue:114]: Is this experiment disabled?

Comment (dyersituations) [reply src/components/WwwFrame/TheHeader.vue:114]: Yes it's currently disabled, but we'll enable it for this new variation.

Comment (dyersituations) [reply src/components/WwwFrame/TheHeader.vue:114]: It's not enabled on dev.

**Comment by roger-in-kiva** [comment]
Looks like login button is a little bit tight... is that okay?

Screenshot 2023-05-15 at 14 35 26

**Comment by dyersituations** [comment]
> Looks like login button is a little bit tight... is that okay?
> 
> Screenshot 2023-05-15 at 14 35 26

Good call, but what resolution is that? How does it look on iPhone setting?

**Comment by roger-in-kiva** [comment]
> > Looks like login button is a little bit tight... is that okay?
> > Screenshot 2023-05-15 at 14 35 26
> 
> Good call, but what resolution is that? How does it look on iPhone setting?

Yeah is for iphone 12 setting
Screenshot 2023-05-15 at 14 54 03

**Comment by dyersituations** [comment]
> Yeah is for iphone 12 pro setting

Ah good to know. Maybe we can reduce some of the padding between the items?

**Comment by roger-in-kiva** [comment]
> > Yeah is for iphone 12 pro setting
> 
> Ah good to know. Maybe we can reduce some of the padding between the items?

I think so, have a smaller gap there and maybe consider to decrease space between logo and Categories menu. For logged in users, looks fine but we are not showing About menu and some hidden icons. So, ensure there is a correct spacing for both cases.

Screenshot 2023-05-15 at 15 03 05

**Comment by roger-in-kiva** [comment]
Looking good for logged out users!
Screenshot 2023-05-18 at 9 09 49

But seems like for logged in users gap is smaller than screenshots above becoming a non-finger friendly ui...
Screenshot 2023-05-18 at 9 13 18

**Pull request merged**